training/training.py

```diff
@@ -195,14 +195,13 @@ def prepare_dataset(command_list, data_dir, input_shape):
                     target_command = command
                     break
 
-            if target_command == "unknown":
-                continue;
-
             original_data = np.loadtxt(os.path.join(subdir, file))
 	    if len(original_data) < 4000:
                 total_pad = 4000 - len(original_data);
 	        left = np.random.randint(total_pad);
```

Comment:
Can you fix up the padding here. Also please update command list arguments so that we don't have to explicitly pass in `unknown` as one of our command